handlers/handlers.go

```diff
@@ -83,14 +87,15 @@ func GeographyRender(rend RenderClient) http.HandlerFunc {
 				"geographyTypesTest": codelistresults.Items[i],
 			})
 
-			geographyTypes = geographyTypes + codelistresults.Items[i].Name
+			geographyTypes = geographyTypes + codelistresults.Items[i].Label
 
 		}
 
 		page.Data.AreaTypes = []geographyHomepage.AreaType{
-			{Name: "Countries"},
-			{Name: "Regions"},
-			{Name: geographyTypes},
+			{Label: "Countries", ID: "country"},
```

Comment:
not entirely clear on this bit, is it still hardcoded and not using the API response?